spec/lib/rbac/query_shared_resource_spec.rb

```diff
@@ -0,0 +1,75 @@
+describe RBAC::QuerySharedResource do
+  let(:app_name) { 'catalog' }
+  let(:resource) { "portfolios" }
+  let(:verbs) { ["read"] }
+  let(:resource_id1) { "1" }
+  let(:role1_uuid) { "1" }
```

Comment:
Are this and `role2_uuid` really buying us much? We're only using it in two places and neither of those are in any expectations/tests, just other lets. I'd argue we should just set the `:uuid` parameter in the following `let`s hardcoded and if for some reason this file grows to include more roles that need to use the value, then we pull it out again.